src/binaryen-c.h

```diff
@@ -2097,6 +2097,15 @@ BINARYEN_API void BinaryenSetMemory(BinaryenModuleRef module,
                                     BinaryenIndex numSegments,
                                     uint8_t shared);
 
+BINARYEN_API int BinaryenHasMemory(BinaryenModuleRef module);
+BINARYEN_API size_t BinaryenMemoryGetInitial(BinaryenModuleRef module);
```

Comment:
```suggestion
BINARYEN_API BinaryenIndex BinaryenMemoryGetInitial(BinaryenModuleRef module);
```

Reply:
I seem to remember initially trying that to fit the other functions but I got a compiler error saying it didn't match the size_t in the .cpp file (which also matched what I was trying to emulate, as I recall, but which compiled without error which confused me)

Reply:
Looks like I did something wrong earlier. :D Changed them to BinaryenIndex and it's looking happy this time. :D